libclamav/crtmgr.h

```diff
@@ -66,8 +66,8 @@ typedef struct cli_crt_t {
     BIGNUM *n;
     BIGNUM *e;
     BIGNUM *sig;
-    time_t not_before;
-    time_t not_after;
+    uint64_t not_before;
+    uint64_t not_after;
```

Comment:
time_t always was defined as signed int, which is why it rolls over at 0x7fffffff (for 32bit) - was the change to unsigned intentional here? The `>> 1` below seems to still account for the sign bit.

Reply:
Fair I guess the `>> 1` wouldn't be necessary since switching to unsigned.  I don't think there's any point in it being signed. (?)

Reply:
There could be compatibility reasons. And there is really no point in being unsigned as we don't need the extra bit.

Reply:
Fair doesn't hurt to use `int64_t`. I just force-pushed the change.